### DISCUSSION TITLE
💛 Testimonials

### DISCUSSION THREAD

**Discussion by yoannfleurydev** [BearStudio/start-ui-web#399] [General]
If you or your company are using 🚀 Start UI [web], we would love to see some feedbacks, comments and your feelings about the project.

We love our community, we hope our community love our work and we hope we can show the world your feedbacks.

Here is a template to simplify your input:

```
- Company/Organization:
- Person name:
- Product/Project:
- Link to the project:
- I agree to be on the https://start-ui.com website in the future testimonials section: ✅ yes / ❌ no
- My feedback:
```

Feel also free so share screenshot of your project if possible, we would love to see what you are building.

**Comment by Billyroot** [comment]
- Company/Organization: Faraday
- Product/Project: Ares Chat /Ares Creative
- Link to the project: https://openares.net  / https://arescreative.fr
- I agree to be on the https://start-ui.com website in the future testimonials section: ✅ yes / 
- My feedback: when we started Ares project one of our main issue was to build appealing UI for our end users on web and Android Platforms. With Start UI figma file we have been able to quickly prototype our Idea, get beta testers feedbacks and even deploy a live version within days instead of weeks. I definetely believe Start UI is a game changer if you want to ship your software quickly with a modern UI from one of the best designers teams in the World.

**Reply by yoannfleurydev** [reply]
Thanks a lot, your testimonial is live on the website! https://start-ui.com/

**Comment by cdue** [comment]
- Company/Organization: Ysely (Freelancer)
- Person name: Cédric Dué
- Product/Project: Private project for one of my clients
- Link to the project: Not public
- I agree to be on the https://start-ui.com website in the future testimonials section: ✅ yes
- My feedback: We used "Start UI" to bootstrap a React project quickly rather than going through the process of setting up the frontend stack (UI, API client, Tests, Design system, and so on). It is a good boilerplate for fast setup. It is opinionated but knowing that the people in charge of this project do a great job at testing/validating libraries, we were really confident. Another important thing about "Start UI" is that it doesn't lock you up as you can remove everything you don't need and change which library you want for another. I will use it again for future projects for sure.

**Reply by yoannfleurydev** [reply]
Thanks a lot, your testimonial is live on the website! https://start-ui.com/
